### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by Reoverv**

### CODE REVIEW

File: src/cmd/api/Authentication.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import bcrypt
+import csv
+
+
+def register(username, password):
```
Comment (ManoloTonto1) [inline src/cmd/api/Authentication.py:5]: unit test these methods.

File: src/cmd/api/Authentication.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+        with open("../../core/storage/users.csv", 'a', newline='') as writeFile:
+            writer = csv.writer(writeFile)
+            salt = bcrypt.gensalt()
+            hashed_password = bcrypt.hashpw(password.encode('utf-8'), salt)
+            writer.writerow([username, hashed_password])
+
+    return True
+
+def Login(username, password):
```
Comment (ManoloTonto1) [inline src/cmd/api/Authentication.py:26]: you should maybe return the JSON token from here. maybe a cleaner solution instead of returning a boolean.

**Review by ManoloTonto1 — CHANGES_REQUESTED** [review]

File: dependencies.txt [hunk 3]
```diff
@@ -1,7 +1,11 @@
 numpy
 scikit-learn
 joblib
+flask
+pandas
+bcrypt
+PyJWT
+dotenv
 peewee
 flask
```
Comment (ManoloTonto1) [inline dependencies.txt:10]: duplicate flask

**Review by Reoverv — COMMENTED** [review]
- Updated dependencies.txt
- will write unit test tonight or tommorow

**Review by ManoloTonto1 — APPROVED** [review]
Make your Unit tests before merging.

**Pull request merged**